mctx/_src/policies.py

```diff
@@ -293,6 +289,11 @@ def stochastic_muzero_policy(
     search tree.
   """
 
+  num_actions = root.prior_logits.shape[-1]
+  num_chance_outcomes = decision_recurrent_fn(
```

Comment:
You can get the num_chance_outcomes from the existing decision_recurrent_fn call, at line 312.